hm-basic-webapp/templates/hpa.yaml

```diff
@@ -1,26 +1,21 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
-  name: {{ include "hm-basic-webapp.prefix" . }}
+  name: {{ include "hm-basic-webapp.fullname" . }}
```

Comment:
I suggest to not change this, `hm-basic-webapp.prefix` is used in other resources in this chart